http_rules/test_http_tables.py

```diff
@@ -430,16 +432,18 @@ def start_all(self):
         self.assertTrue(srv.wait_for_connections(timeout=1))
 
     def test(self):
+        if type(self) is HttpTablesTestBase:
+            self.skipTest("This is an abstract class")
```

Comment:
I guess the problem was that the framework calls all `test()` methods of all classes. `cache/test_cache_control.py` uses `_test()` method and `SingleTest` wrapping class for this.

Could you please pick one of the approaches and move it somewhere it can be used by all the tests, e.g. to `helpers/util.py`. All the tests (there are not so many though) must be updated. Consider also https://github.com/tempesta-tech/tempesta-test/pull/219/files#diff-7f132ed50763eb65b8bfe8ed655fb26af29ef9cf95c5c602cfc1d91c32e37b3fR87

Reply:
Done in #235